src/main/java/uk/gov/hmcts/juror/api/bureau/service/BureauAuthenticationServiceImpl.java

```diff
@@ -52,6 +52,7 @@ public BureauAuthenticationController.BureauAuthenticationResponseDto authentica
                     999,
                     false,
                     user.getUserType(),
+                    user.getUserType(),
```

Comment:
does this need to be getActiveUserType() ?

Reply:
no because active user type is non existent. in the new authentication the active user type is set based on the type of court the user is viewing as... so admin viewing as court would be set as court and so on.